bazel/repository_locations.bzl

```diff
@@ -34,8 +34,8 @@ REPOSITORY_LOCATIONS = dict(
         remote = "https://github.com/grpc/grpc.git",
     ),
     io_opentracing_cpp = dict(
-        commit = "e57161e2a4bd1f9d3a8d3edf23185f033bb45f17",
-        remote = "https://github.com/opentracing/opentracing-cpp", # v1.2.0
+        commit = "1368b7de0343388a0e217643481ec4d475ed22e5",
+        remote = "https://github.com/rnburn/opentracing-cpp",
```

Comment:
do you know when https://github.com/opentracing/opentracing-cpp/pull/45 will merge?

Reply:
I'm hoping to get it merged in this week.